src/features/shared-link-settings-modal/ExpirationSection.js

```diff
@@ -29,6 +29,7 @@ const ExpirationSection = ({
     const datepicker = (
         <div>
             <DatePicker
+                dateFormat="utcTime"
```

Comment:
the default date is using the browser's timezone: https://github.com/box/box-ui-elements/blob/1ab5eb11137ba471039f1b6c4bc8119358bf1de0/src/features/shared-link-settings-modal/SharedLinkSettingsModal.js#L146